ci_framework/roles/edpm_build_images/tasks/install.yml

```
@@ -0,0 +1,25 @@
+---
+- name: Add edpm-image-builder to the list of packages to be installed on host
+  set_fact:
+    cifmw_build_host_packages: "{{ cifmw_build_host_packages + ['edpm-image-builder'] }}"
```

```suggestion
    cifmw_edpm_build_images_host_packages: "{{ cifmw_edpm_build_images_packages_host_packages + ['edpm-image-builder'] }}"
```

Note: with the length of the parameter name, we MAY use `cifmw_ebi_` prefix instead.